### ISSUE TITLE
Splunk Windows pod failure  

### ISSUE THREAD

**Issue by ssuzze** [signalfx/splunk-otel-collector-chart#459]
We have installed chart=splunk-otel-collector-0.51.0 on EKS windows node cluster and encountering pod failures per logs:

Trace[………]: ---"Objects listed" error:Get "https://......../api/v1/pods?fieldSelector=spec.nodeName%3Dip-............&limit=500&resourceVersion=0": dial tcp……. connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. 21002ms (…..)

with values:

logsEngine: otel
cloudProvider: "aws"
distribution: "eks"

  securityContext:
    windowsOptions:
      runAsUserName: "ContainerUser"

clusterReceiver:
  enabled: true

readinessProbe:
  initialDelaySeconds: 60
livenessProbe:
  initialDelaySeconds: 60

isWindows: true

and made changes to the linux value file as suggested per Advance Configuration instructions. 

I appreciate any input to this Pod log behavior, thank you.

**Comment by matthewmodestino** [comment]
Can you review this previous issue I opened covering some of what I had to do to get it deployed?

https://github.com/signalfx/splunk-otel-collector-chart/issues/428

Tldr, had to use containerAdmin and change liveness probes as per notes in helm chart.

**Comment by atoulme** [comment]
Closing as the bug has been inactive for a few months. Please reopen or comment if more work is needed.
